multicluster/charts/linkerd-multicluster/Chart.yaml

```diff
@@ -11,7 +11,7 @@ kubeVersion: ">=1.21.0-0"
 name: "linkerd-multicluster"
 sources:
 - https://github.com/linkerd/linkerd2/
-version: 30.6.6-edge
+version: 31.0.0
```

Comment:
What's the reason for the major bump?

Reply:
i think i misinterpreted how the version rules apply to https://github.com/linkerd/linkerd2/commit/371e27e73b1927964a14ec204572ebb21e3e4cec, it should have been minor i think?